package core.exceptions;

public class NameAlreadyInUseException extends Exception{

}
